### PR TITLE
Main Menu Responsive with total NFTs from the proxy endpoint

### PR DESCRIPTION
**Pull request by miryamfv**

### CODE REVIEW

**Comment by miryamfv** [comment]
In this PR 
1. Moved the connect button to outside the top menu
2. Added responsive dropdown
3. Styles are unfished in jus sendinf the functionality

File: components/layout.js [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
 import TopMenu from './top-menu'
 import PropTypes from 'prop-types'
+import MainMenu from './main-menu'
+import { MenuEventContext } from '../context/menu-event-context'
+import { UserContext } from '../context/user-context'
+
 
 export default function Layout({ children }) {
+    const user = useContext(UserContext)
+    const { onConnectWallet, onToggleMenu, setToggleMenuAction } = useContext(MenuEventContext)
```
Comment (lorenzopolidori) [inline components/layout.js:15]: `onToggleMenu` is not used anymore. You should be able to see these issues by running `npm run lint`.

**Review by lorenzopolidori — CHANGES_REQUESTED** [review]
There are a few `eslint` issues.
You can check them by running `npm run lint`.

VSCode can also help you with linter issues. If you install the ESLint extension, you will see the issues inline, see screenshot:

image


There are also a few formatting issues. You can fix them by re-saving your changes in each file after you have set up your VSCode with "Format on Save" enabled, see below.

image

You can also use prettify. I am not using it, but you can ask @masafumimori, as he added rules in the repo and I think he is using it.

File: .gitignore [hunk 2]
```diff
@@ -28,3 +28,4 @@ npm-debug.log*
 .env.development.local
 .env.test.local
 .env.production.local
+yarn.lock
```
[the remark this answers is not in the document]
Comment (miryamfv) [reply .gitignore:31]: I used yarn for plugins needed for Babylon. I thought it needed to be ignored I can inignore

File: package-lock.json [hunk 3]
```diff
@@ -24199,6 +24214,12 @@
       "resolved": "https://registry.npmjs.org/yargs-parser/-/yargs-parser-21.0.1.tgz",
       "integrity": "sha512-9BK1jFpLzJROCI5TzwZL/TU4gqjK5xiHV/RfWLOahrjAko/e4DJkRDZQXfvqAsiZzzYhgAzbgz6lg48jcm4GLg==",
       "dev": true
+    },
+    "@next/swc-freebsd-x64": {
+      "version": "12.2.2",
+      "resolved": "https://registry.npmjs.org/@next/swc-freebsd-x64/-/swc-freebsd-x64-12.2.2.tgz",
+      "integrity": "sha512-JUnXB+2xfxqsAvhFLPJpU1NeyDsvJrKoOjpV7g3Dxbno2Riu4tDKn3kKF886yleAuD/1qNTUCpqubTvbbT2VoA==",
+      "optional": true
     }
```
[the remark this answers is not in the document]
Comment (miryamfv) [reply package-lock.json:24223]: Mm I reverted changes made when installing Babylon not sure why package.json didn't. I can revert again.would it be better to disregard this whole PR and re do from the fresh clone that I have to do?

[the remark this answers is not in the document]
Comment (lorenzopolidori) [reply package-lock.json:24223]: Ok with me

**Pull request closed without merging**